README.md

```diff
@@ -410,6 +411,50 @@ See [BIGTREETECH-TouchScreenFirmware/releases](https://github.com/bigtreetech/BI
 
 
 ## Appendix
+### A basic WIP documentation for RRF users
```

Comment:
I would exclude this line, and make the next line as the title (TFT Screen Configuration and support for RRF) but otherwise great, thank you! :)

Reply:
Done